styles/_myft-card.scss

```diff
@@ -19,15 +19,23 @@
 
 .topic-card__myft-header {
 	display: flex;
-	background: getColor('claret');
 	padding: 0 10px;
 }
 
 .topic-card__myft-title {
 	@include oTypographySansBold($scale: 2, $progressive: true);
-	color: getColor('white');
+	color: getColor('black');
 	flex: 1 1 auto;
 	padding-right: 10px;
+
+	&:after {
+		content: '';
+		display: block;
+		width: 85px;
+		height: 5px;
+		@include oTypographyMargin($top: 2);
+		background: oColorsGetPaletteColor('black');
```

Comment:
Could we just use ` getColor('black');` here? (And if not, why not, seeing as it works for the `.topic-card__myft-header` and `.topic-card__myft-title` classes)

Reply:
Yep, will update. This was just lazy copy pasting from [the search page](https://github.com/Financial-Times/next-search-page/blob/master/client/components/_filters.scss#L13-L19)!